models/forward_tacotron.py

```diff
@@ -234,29 +295,27 @@ def generate(self,
         x_post = x_post.cpu().data.numpy()
         dur = dur.cpu().data.numpy()
 
-        return x, x_post, dur, pitch_hat
+        return {'mel': x, 'mel_post': x_post, 'dur': dur,
+                'pitch': pitch_hat, 'energy': energy_hat}
 
-    def pad(self, x, max_len):
+    def pad(self, x: torch.tensor, max_len: int) -> torch.tensor:
         x = x[:, :, :max_len]
         x = F.pad(x, [0, max_len - x.size(2), 0, 0], 'constant', -11.5129)
```

Comment:
The hardcoded value here is different in sign from the previous one, is this wanted?
Also I'd just either set it as a class attribute or config it